src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -115,6 +115,13 @@
       + "is automatically destroyed.";
   public static final String CONSUMER_INSTANCE_TIMEOUT_MS_DEFAULT = "300000";
 
+  public static final String SIMPLE_CONSUMER_POOL_SIZE_CONFIG = "simpleconsumer.pool.size";
+  private static final String
+      SIMPLE_CONSUMER_POOL_SIZE_DOC =
+      "maximum number of SimpleConsumers that can be instantiated by broker."
+      + " If empty, then no pool is used.";
```

Comment:
See notes on the SimpleConsumerPool classes, but I'm pretty sure this is not the behavior we want. "If 0, then the pool size is not limited." is probably what we want.